test/model/binding.dart

```diff
@@ -557,27 +528,28 @@ class FakeFlutterLocalNotificationsPlugin extends Fake implements FlutterLocalNo
 }
 
 class FakeAndroidFlutterLocalNotificationsPlugin extends Fake implements AndroidFlutterLocalNotificationsPlugin {
+}
+
+class FakeIOSFlutterLocalNotificationsPlugin extends Fake implements IOSFlutterLocalNotificationsPlugin {
```

Comment:
nit: these can be cut too, since they're effectively part of the implementation of `resolvePlatformSpecificImplementation`

(the previous revision did cut them, so I'm guessing their coming back was a rebase error)